asmcli/components/ca/managed-cas.sh

```diff
@@ -108,23 +108,21 @@ x_wait_for_enabling_workload_certificates() {
   # Enabling workload certificate feature on a cluster will result in
   # the cluster restarting and the cluster becoming unreachable for minutes.
   # The cluster restart may happen at a random time after the enablement.
-  # Therefore, wait 15 minutes before checking the connection to k8s is recovered.
```

Comment:
Why this wait is removed?

Reply:
since we are waiting until spiffe-node-agent is installed, I thought this check is unnecessary. the time has instead been moved to the loop below. WDYT?

Reply:
Moving the wait here to waiting until spiffe-node-agent is installed does not always work because the cluster restart may happen at a random time after the enablement. For example, if the  wait here is moved to waiting until spiffe-node-agent, when waiting for spiffe-node-agent, the cluster may have not restarted yet, thus spiffe-node-agent check may pass and the installation may fail at a later installation stage due to the cluster restart, which explains why the output from Lingqing's test is not the same as that of yours.

Reply:
added back wait